### ISSUE TITLE
Fix scale of number to px

### ISSUE THREAD

**Issue by fcisio** [atanasster/babel-plugin-theme-ui#3]
The numbers values don't seem to be transformed to pixel values like Theme UI does.

Example:

```js
space: [0, 4, 8, 16, 32, 64, 128, 256, 512],
//

// input
a: {
  "--test": "space.4",
},

// output
a: {
  "--test": 32,

  // Should be 
  "--test": "32px",
},
```

![Shot_0 5](https://user-images.githubusercontent.com/46095852/99724639-ea00c980-2a81-11eb-8e53-2823d81bbb5a.jpg)

**Comment by atanasster** [comment]
- on one hand `space: ['0px', '4px', '8px', '16px', '32px', '64px', etc],` should work.
- we can add space formatting settings, something like
```
  spaceFormats: {
    'space': value => `${value}px`,
  }
```
this should be more versatile but might over complicate the settings

what do you think we should do here?

**Comment by fcisio** [comment]
Personally, I would not transform numbers to px, I don't mind just using string px values ("40px").

But this is not consistent with the Theme UI which transforms those values [See Doc](https://theme-ui.com/guides/object-styles#numbers). That means that the plugin will break the themes.

We can either:
- Transform those value to prevent the themes from breaking
- Add a disclaimer in the readme saying that pixel values need to be entered as strings. Which make it simpler and probably more performant

**Comment by atanasster** [comment]
thanks a lot, released 0.6.0 with `spaceFormats` option - by default it will transform values into the `space` key.

closing this issue for now
